### DISCUSSION TITLE
Can we load trait on editor load but without clicking components ?

### DISCUSSION THREAD

**Discussion by iabhiyaan** [GrapesJS/grapesjs#3539] [Q&A]
Hello @artf, Is there any way to load traits in trait manager but without clicking on components ?

**Comment by artf** [comment]
Traits show details about the selected component so what is the point?
The only way would be to select a component on editor load.

**Reply by iabhiyaan** [reply]
There is requirement that menu related trait should be shown on load. That's why. Thanks for your help.

**Reply by artf** [reply]
Ok, that means you have to select the menu on load otherwise you won't be able to interact with those traits

**Reply by iabhiyaan** [reply]
@artf, can we render trait at different places. I mean to say if we do appendTo:'#menuTrait'. All traits will be rendered there.It's fine for menu.
However, could it be possible to render some other section's trait to another id. E.g blog should be rendered on another place ('#blogTrait').Could we achieve that?
